test/metabase/test_runner/assert_exprs.clj

```diff
@@ -24,44 +23,19 @@
        form))
    form))
 
-(declare ^:private strip-generated-idents)
```

Comment:
Updated this to remove all idents entirely from both `expected` and `actual`. This is temporary until https://github.com/metabase/metabase/pull/61310 lands